src/vm/docs/mod.rs

```diff
@@ -489,6 +489,38 @@ input list, and returns the same list with any elements removed for which the `f
     example: "(filter not (list true false true false)) ;; Returns (false false)"
 };
 
+const BUFF_TO_INT_LE_API: SpecialAPI = SpecialAPI {
+    input_type: "(buff 16)",
+    output_type: "int",
+    signature: "(buff-to-int-le (buff 16))",
+    description: "Converts a 16-byte buffer to a signed integer use a little-endian encoding.",
+    example: "",
```

Comment:
It'd be good to include some examples for these functions -- they'd double as tests as well.

Reply:
When I try to add the examples, they fail with the error

`err: NonFunctionApplication, expressions: Some([SymbolicExpression { expr: Atom(ClarityName("buff-to-int-le"))...`

Is this a clarity version issue?

Reply:
There shouldn't be a versioning issue -- what's the example that you're trying to add?

Reply:
It was a version issue.. there was a Clarity1 assumption in the test.. I changed it and added a bunch of examples.